CRVReco/fcl/prolog_v11.fcl

```diff
@@ -5,12 +5,14 @@ BEGIN_PROLOG
       module_type               : CrvPedestalFinder
       crvDigiModuleLabel        : "CrvDigi"
       maxADCspread              : 5
+      tmpDBfileName             : "pedestals.txt"
```

Comment:
How temporary are these files? Will they be replaced with database content eventually? Is there a reason you can't use the database text file override and need to enter them as explicit parameters here?

Reply:
These are the names of the DB text files that are being produced by the pedestal finder and calibration module. The file produced by the pedestal finder will only be used by the calibration module and can be discarded afterwards. The file produced by the calibration module needs to go through the validation step. Once it has been verified that the content of this file (i.e. the pedestals and calibration constants) is Ok, it can be submitted to the official database and discarded afterwards.